content/docs/user-guide/how-to/best-practices.md

```diff
@@ -0,0 +1,136 @@
+# Best Practices for DVC Projects
+
+Asking questions on data science collaboration to data scientists, engineers, or
+managers, we'll get a variety of answers. DVC provides a systematic approach
+towards managing and collaborating on data science projects. You can manage your
+projects with DVC more efficiently using the practices listed here:
+
+- Source code and data versioning
+
+  You can use DVC to avoid discrepancies between
+  [revisions](https://git-scm.com/docs/revisions) of source code and versions of
+  data files, when the data doesn't fit into a traditional repository. DVC
+  replaces all large data files, models, etc. with small
+  [metafiles](doc/user-guide/dvc-files-and-directories) (tracked by Git). These
+  files point to the original data, which you can access by checking out the
+  required `revision`.
+
+- Experiments
+
+  You can make use of Git branches to document progress of training different
+  types of models on your data files in the same project. Create a branch for
+  each of the models and then utilise DVC features while working on that branch.
+
+- Experiment time log
+
+  [Hyperparameter](<https://en.wikipedia.org/wiki/Hyperparameter_(machine_learning)>)
+  are defined using the the `--params` option of `dvc run` and the default
+  parameters file is `params.yaml`. You can commit different versions of
```

Comment:
Please introduce the problem first because it's not obvious from the title. In fact I don't really love this title... Should maybe use both "experiment" and "parameter" terms?

Reply:
"defined using the the `--params` option of `dvc run` and the default parameters file is `params.yaml`." - too much information. Try to simplify and mention only the absolutely necessary. Better to link to the params reference for the rest of the details.